microsoft/PowerToys
# [FindMyMouse] Add setting for excluded apps

## Summary of the Pull Request

**What is this about:**
Game mode doesn't detect all the apps users want to avoid activating Find My Mouse in.
Adding an excluded apps list would be way for users to define these apps.

**What is included in the PR:** 
Excluded apps functionality.
Add setting for editing excluded apps.
Refactor a common function.

![image](https://user-images.githubusercontent.com/26118718/153720583-527657fc-a058-4646-8cc0-bb71a469c15a.png)

**How does someone test / validate:** 
Add some apps to the excluded apps in settings and verify FindMyMouse doesn't activate with that app in the foreground.

## Quality Checklist

- [x] **Linked issue:** #14071 #15856
